cygnus-common/README.md

```diff
@@ -33,7 +33,7 @@ Simply configure the FIWARE repository if not yet configured:
     $ cat > /etc/yum.repos.d/fiware.repo <<EOL
     [Fiware]
     name=FIWARE repository
-    baseurl=http://repositories.testbed.fi-ware.eu/repo/rpm/x86_64/
+    baseurl=http://repositories.testbed.fiware.eu/repo/rpm/x86_64/
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1375/commits/b0b2e859498b6c1a20112df2a0b373f9b41dade4